diff --git a/ui/build/dumpvars.go b/ui/build/dumpvars.go
index 5a104480a..18ca15966 100644
--- a/ui/build/dumpvars.go
+++ b/ui/build/dumpvars.go
@@ -139,7 +139,7 @@ func dumpMakeVars(ctx Context, config Config, goals, vars []string, write_soong_
 var BannerVars = []string{
 	"PLATFORM_VERSION_CODENAME",
 	"PLATFORM_VERSION",
-	"CR_VERSION",
+	"FLOKO_VERSION",
 	"PRODUCT_INCLUDE_TAGS",
 	"TARGET_DEVICE",
 	"TARGET_BUILD_VARIANT",
