diff --git a/res/drawable/logo.png b/res/drawable/logo.png
old mode 100755
new mode 100644
index e34b045..53492db
Binary files a/res/drawable/logo.png and b/res/drawable/logo.png differ
diff --git a/res/layout/setup_lineage_settings.xml b/res/layout/setup_lineage_settings.xml
index 5d206e4..467c03a 100644
--- a/res/layout/setup_lineage_settings.xml
+++ b/res/layout/setup_lineage_settings.xml
@@ -46,6 +46,7 @@
                     android:layout_height="match_parent">
 
                     <!-- Whether or not to enable metrics -->
+<!-- disable it!
                     <LinearLayout
                         android:id="@+id/metrics"
                         android:orientation="horizontal"
@@ -73,6 +74,7 @@
                             style="@style/SudCheckBox.Multiline" />
 
                     </LinearLayout>
+-->
 
                     <!-- Checkbox for using on-screen nav keys -->
                     <LinearLayout
diff --git a/src/org/lineageos/setupwizard/LineageSettingsActivity.java b/src/org/lineageos/setupwizard/LineageSettingsActivity.java
index defa0a0..3aa49ce 100644
--- a/src/org/lineageos/setupwizard/LineageSettingsActivity.java
+++ b/src/org/lineageos/setupwizard/LineageSettingsActivity.java
@@ -18,7 +18,6 @@
 package org.lineageos.setupwizard;
 
 import static org.lineageos.setupwizard.SetupWizardApp.DISABLE_NAV_KEYS;
-import static org.lineageos.setupwizard.SetupWizardApp.KEY_SEND_METRICS;
 
 import android.app.Activity;
 import android.content.Context;
@@ -46,21 +45,14 @@ public class LineageSettingsActivity extends BaseSetupWizardActivity {
 
     public static final String TAG = LineageSettingsActivity.class.getSimpleName();
 
-    public static final String PRIVACY_POLICY_URI = "https://crdroid.net/legal";
+    public static final String PRIVACY_POLICY_URI = "https://wiki.maud.io/floko/legal";
 
     private SetupWizardApp mSetupWizardApp;
 
-    private CheckBox mMetrics;
     private CheckBox mNavKeys;
 
     private boolean mSupportsKeyDisabler = false;
 
-    private final View.OnClickListener mMetricsClickListener = view -> {
-        boolean checked = !mMetrics.isChecked();
-        mMetrics.setChecked(checked);
-        mSetupWizardApp.getSettingsBundle().putBoolean(KEY_SEND_METRICS, checked);
-    };
-
     private final View.OnClickListener mNavKeysClickListener = view -> {
         boolean checked = !mNavKeys.isChecked();
         mNavKeys.setChecked(checked);
@@ -80,20 +72,6 @@ public class LineageSettingsActivity extends BaseSetupWizardActivity {
                 privacyPolicy, policySummary);
         getGlifLayout().setDescriptionText(servicesFullDescription);
 
-        View metricsRow = findViewById(R.id.metrics);
-        metricsRow.setOnClickListener(mMetricsClickListener);
-        metricsRow.requestFocus();
-        String metricsHelpImproveLineage =
-                getString(R.string.services_help_improve_cm, os_name);
-        String metricsSummary = getString(R.string.services_metrics_label,
-                metricsHelpImproveLineage, os_name, os_name);
-        final SpannableStringBuilder metricsSpan = new SpannableStringBuilder(metricsSummary);
-        metricsSpan.setSpan(new android.text.style.StyleSpan(android.graphics.Typeface.BOLD),
-                0, metricsHelpImproveLineage.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
-        TextView metrics = (TextView) findViewById(R.id.enable_metrics_summary);
-        metrics.setText(metricsSpan);
-        mMetrics = (CheckBox) findViewById(R.id.enable_metrics_checkbox);
-
         View navKeysRow = findViewById(R.id.nav_keys);
         navKeysRow.setOnClickListener(mNavKeysClickListener);
         mNavKeys = (CheckBox) findViewById(R.id.nav_keys_checkbox);
@@ -110,7 +88,6 @@ public class LineageSettingsActivity extends BaseSetupWizardActivity {
     public void onResume() {
         super.onResume();
         updateDisableNavkeysOption();
-        updateMetricsOption();
     }
 
     @Override
@@ -134,15 +111,6 @@ public class LineageSettingsActivity extends BaseSetupWizardActivity {
         return R.drawable.ic_features;
     }
 
-    private void updateMetricsOption() {
-        final Bundle myPageBundle = mSetupWizardApp.getSettingsBundle();
-        boolean metricsChecked =
-                !myPageBundle.containsKey(KEY_SEND_METRICS) || myPageBundle
-                        .getBoolean(KEY_SEND_METRICS);
-        mMetrics.setChecked(metricsChecked);
-        myPageBundle.putBoolean(KEY_SEND_METRICS, metricsChecked);
-    }
-
     private void updateDisableNavkeysOption() {
         if (mSupportsKeyDisabler) {
             final Bundle myPageBundle = mSetupWizardApp.getSettingsBundle();
diff --git a/src/org/lineageos/setupwizard/util/SetupWizardUtils.java b/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
index b3d49fd..591f3b8 100644
--- a/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
+++ b/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
@@ -71,7 +71,7 @@ public class SetupWizardUtils {
     private static final String GMS_PACKAGE = "com.google.android.gms";
     private static final String GMS_SUW_PACKAGE = "com.google.android.setupwizard";
     private static final String GMS_TV_SUW_PACKAGE = "com.google.android.tungsten.setupwraith";
-    private static final String UPDATER_PACKAGE = "com.crdroid.updater";
+    private static final String UPDATER_PACKAGE = "io.maud.floko.updater";
 
     private static final String UPDATE_RECOVERY_EXEC = "/vendor/bin/install-recovery.sh";
     private static final String CONFIG_HIDE_RECOVERY_UPDATE = "config_hideRecoveryUpdate";
